cfgov/jinja2/v1/_includes/templates/header.html

```diff
@@ -31,21 +31,16 @@
 
     {% include 'molecules/global-search.html' with context %}
 
-    <a href="/">
-        <img class="o-header_logo header_logo u-js-only"
-             src="/static/img/logo_sm-exec.svg"
-             onerror="this.onerror=null;this.src='/static/img/logo_sm-exec.png';"
-             alt="Consumer Financial Protection Bureau"
-             width="237"
-             height="50">
-        <noscript>
-            <img class="header_logo logo__no-js"
-                 src="/static/img/logo_sm-exec.png"
+        <a class="o-header_logo" href="/">
+            <img class="o-header_logo-img u-js-only"
+                 src="/static/img/logo_sm-exec.svg"
+                 onerror="this.onerror=null;this.src='/static/img/logo_sm-exec.png';"
                  alt="Consumer Financial Protection Bureau"
                  width="237"
                  height="50">
             <noscript>
-                <img class="header_logo logo__no-js"
+                <img class="o-header_logo-img
+                            o-header_logo-img__no-js"
```

Comment:
Removed modifier in https://github.com/cfpb/cfgov-refresh/commit/7130a237315ae210726d0d29a0d05cd2b5b32728